### PR TITLE
table_advansed

### PR DESCRIPTION
**Pull request by MaksymKos**
https://maksymkos.github.io/react_people-table-advanced/

### CODE REVIEW

File: src/components/PeopleFilters.tsx [hunk 1]
```diff
@@ -1,12 +1,50 @@
...
+import { SearchLink } from './SearchLink';
+import { getSearchWith } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const centuriesList = ['16', '17', '18', '19', '20'];
+
+  const sex = searchParams.get('sex');
+  const centuries = searchParams.getAll('centuries');
+  const query = searchParams.get('query') || '';
```
Comment (VitaliyBondarenko1982) [inline src/components/PeopleFilters.tsx:13]: suggest to make enum for all possible searchParams

**Review by VitaliyBondarenko1982 — CHANGES_REQUESTED** [review]
Looks good.
Compare your implementation with working example.
А couple mismatches founded.
- when exists some searchParams and click on person link - searchParams shouldn't be resetted
- when order in desc direction- should be visible only bottom arrow 
actual:
actual
working example:
working_example

File: src/components/PeoplePage.tsx [hunk 2]
```diff
@@ -1,12 +1,102 @@
...
+        {
+          prepared.sort((a, b) => a.name.localeCompare(b.name));
+          break;
+        }
+
+        case 'sex':
+        {
+          prepared.sort((a, b) => a.sex.localeCompare(b.sex));
+          break;
+        }
```
Comment (etojeDenys) [inline src/components/PeoplePage.tsx:74]:
```suggestion
        case 'name':
        case 'sex':
        {
          prepared.sort((a, b) => a[sort].localeCompare(b[sort]));
          break;
        }
```
you can group some cases

**Review by etojeDenys — CHANGES_REQUESTED** [review]
i cannot find PeopleTable component in github

**Review by VitaliyBondarenko1982 — APPROVED** [review]
Well done.

**Pull request closed without merging**